helm/sematic-server/templates/ingress.yaml

```diff
@@ -34,6 +35,17 @@ spec:
                 name: {{ $serviceName }}
                 port:
                   number: {{ $servicePort }}
+          {{- if $.Values.deployment.socket_io.dedicated }}
```

Comment:
I think we would want the socketio instance to be dedicated for socketio broadcasts, and not also receive other API calls. This would also simplify the scaling out of the other server deployment.

Please remove the socketio instance from the rest of the path matching.

Reply:
that is already what is happening here.  there's a single ingress in front of both of the servers, and only api calls go to the api one, and only socketio calls go to socketio.